tools/cli.py

```diff
@@ -344,6 +345,11 @@ def Main() -> None:
             args=[
                 ('path', 'Path to bucket.', None),
             ])
+  AddParser('gcp', gcp_subparsers, 'bucketsize',
+            'Get the size of a GCS bucket.',
+            args=[
+                ('path', 'Path to bucket.', None)
```

Comment:
Should this be `--path` since it's an optional argument of `GetBucketSize`?

Reply:
Since it's now required, I think this makes sense again. I've renamed the argument inside the function, and now this is consistent with the other functions that take in a bucket name (listobjects, bucketacls)